client/dashboard/me/tax-details/compose-analytics.ts

```diff
@@ -0,0 +1,12 @@
+import { flatMap, property } from 'lodash';
```

Comment:
Note: we're trying to keep `lodash` out of the Hosting Dashboard 🙂

Reply:
Removed in 617ac23.